src/test/descriptor_tests.cpp

```diff
@@ -40,32 +43,49 @@ bool EqualDescriptor(std::string a, std::string b)
     return a == b;
 }
 
-std::string MaybeUseHInsteadOfApostrophy(std::string ret)
+std::string UseHInsteadOfApostrophe(std::string ret)
```

Comment:
It's in an anonymous namespace, or am I missing something?

Reply:
True, sorry.

Reply:
But make it `std::string UseHInsteadOfApostrophe(const std::string& ret)`. Also could avoid `ret` argument, sounds like it's an output/return argument.

Reply:
I believe that this line would fail should ret be made a const reference:
https://github.com/bitcoin/bitcoin/blob/0b90954416767f6213094f1f282e708c3e8a710b/src/test/descriptor_tests.cpp#L50
I guess I could change the signature to what you suggest and then assign `ret` to a local variable. Then there would still be a single copy operation so it's not less efficient or anything, but that would introduce an extra line of code. Do you think I should do it?
I agree that `ret` is not an indicative name. I think `desc` is better?